iceoryx_posh/source/popo/ports/interface_port.cpp

```diff
@@ -38,6 +38,7 @@ void InterfacePort::dispatchCaProMessage(const capro::CaproMessage& caProMessage
         // information loss for this interface port
```

Comment:
How is this file related to the design decision document? Doesn't it violate our traceability rule when we change files which are unrelated to the issue?!

Reply:
This is a draft PR and meant to show you what kind of code changes are planned